theano/tensor/basic.py

```diff
@@ -5862,53 +5862,6 @@ def R_op(self, inputs, eval_points):
         if eval_points[0] is None and eval_points[1] is None:
             return [None]
 
-        debugger_available = config.compute_test_value != 'off'
-
-        if debugger_available:
-            try:
-                iv0 = gof.op.get_test_value(inputs[0])
-            except AttributeError:
-                gof.op.missing_test_message(
-                    'first input passed to Dot.R_op has no test value')
-                debugger_available = False
-
-            try:
-                iv1 = gof.op.get_test_value(inputs[1])
-            except AttributeError:
-                gof.op.missing_test_message(
-                    'second input passed to Dot.R_op has no test value')
-                debugger_available = False
-
-            if eval_points[0]:
-                try:
-                    ev0 = gof.op.get_test_value(eval_points[0])
-                except AttributeError:
-                    gof.op.missing_test_message(
-                        'first eval point passed to Dot.R_op '
-                        'has no test value')
-                    debugger_available = False
-            if eval_points[1]:
-                try:
-                    ev1 = gof.op.get_test_value(eval_points[1])
-                except AttributeError:
-                    gof.op.missing_test_message(
-                        'second eval point passed to Dot.R_op '
-                        'has no test value')
-                    debugger_available = False
-
-        if debugger_available:
-            input_values = [iv0, iv1]
-            eval_point_values = [ev0, ev1]
-
-            for i in xrange(2):
-                if eval_point_values[i] is not None and \
-                   input_values[i].shape != eval_point_values[i].shape:
-                    raise ValueError(
-                        'input ' + str(i) + ' and eval_point ' + str(i) +
-                        ' to Dot.R_op should have the same shape, but '
-                        'their shapes are %s and %s, respectively' % (
-                            str(input_values[i].shape),
-                            str(eval_point_values[i].shape)))
```

Comment:
It gave an error about undefined variables and I question its usefulness.

This is the only op to have a weird debug behavior in reaction to compute_test_values, which should anyway catch most of the problem that this code was attempting to report.